### PR TITLE
busco: add tar requirement

### PR DESCRIPTION
**Pull request by bernt-matthias**
From the logs there remains the error: 

```
[makeblastdb]	/usr/local/bin/makeblastdb: error while loading shared libraries: libbz2.so.1
```

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/busco/busco.xml [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+<tool id="busco" name="Busco" version="@TOOL_VERSION@+galaxy2">
     <description>assess genome assembly and annotation completeness</description>
     <macros>
         <token name="@TOOL_VERSION@">3.0.2</token>
     </macros>
     <requirements>
         <requirement type="package" version="@TOOL_VERSION@">busco</requirement>
         <requirement type="package" version="3.5.1">r-base</requirement>
-        <requirement type="package" version="3.3.2">augustus</requirement>
+		<requirement type="package" version="3.3.2">augustus</requirement>
```
Comment (mvdbeek) [inline tools/busco/busco.xml]:
```suggestion
        <requirement type="package" version="3.3.2">augustus</requirement>
```

File: tools/busco/busco.xml [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
+		<requirement type="package" version="1.32">tar</requirement>
```
Comment (mvdbeek) [inline tools/busco/busco.xml]:
```suggestion
        <requirement type="package" version="1.32">tar</requirement>
```

File: tools/busco/busco.xml [hunk 3]
```diff
@@ -1,12 +1,13 @@
-<tool id="busco" name="Busco" version="@TOOL_VERSION@+galaxy1">
+<tool id="busco" name="Busco" version="@TOOL_VERSION@+galaxy2">
```
Comment (mvdbeek) [inline tools/busco/busco.xml]: Can you also add a `profile="18.01"` ... the tool doesn't set `detect_errors="exit_code"` and we'd get this plus isolated home etc., for free.

**Comment by bernt-matthias** [comment]
> busco pins blast to < 2.4, which doesn't seem to ship bzip2 as a dependency --- but bzip2 is listed in https://github.com/bioconda/bioconda-recipes/blob/master/recipes/blast/2.2.21/meta.yaml, which is pretty weird.

seems that the build number was not bumped after the bzip2 requirement was added.

**Comment by mvdbeek** [comment]
I think it's actually 2.2.31 that gets installed, whose last known build looked like this: https://github.com/bioconda/bioconda-recipes/blob/de1a2caef2b4fa26f37e96ddbd404156a5ae2e60/recipes/blast/meta.yaml

**Comment by bernt-matthias** [comment]
This seems related: https://github.com/bioconda/bioconda-recipes/issues/15459

**Comment by bernt-matthias** [comment]
now that https://github.com/bioconda/bioconda-recipes/pull/19016 is merged we could try this one again. seems stuck anyway

**Comment by mvdbeek** [comment]
It passed here, the status just didn't update
It nows also passed in biocontainers:
```
All 5 test(s) executed passed.
busco[0]: passed
busco[1]: passed
busco[2]: passed
busco[3]: passed
busco[4]: passed
```

**Pull request merged**